rules/sonic-utilities.mk

```diff
@@ -5,6 +5,7 @@ $(SONIC_UTILITIES_PY3)_SRC_PATH = $(SRC_PATH)/sonic-utilities
 $(SONIC_UTILITIES_PY3)_PYTHON_VERSION = 3
 $(SONIC_UTILITIES_PY3)_DEPENDS += $(SONIC_PY_COMMON_PY3) \
                                   $(SWSSSDK_PY3) \
+                                  $(SONIC_PLATFORM_COMMON_PY3) \
                                   $(SONIC_CONFIG_ENGINE_PY3) \
                                   $(SONIC_PLATFORM_COMMON_PY3) \
```

Comment:
This PR is unnecessary. I already added the dependency on this line via https://github.com/Azure/sonic-buildimage/pull/6703